build.gradle.kts

```diff
@@ -80,9 +97,53 @@ subprojects {
         }
     }
 
-    apply(plugin = "com.vanniktech.maven.publish")
-    apply(plugin = "org.jetbrains.kotlinx.binary-compatibility-validator")
-    apply(from = "$rootDir/gradle-mvn-publish.gradle")
+    apply(plugin = "com.github.ben-manes.versions")
```

Comment:
Do we need this plugin to switch to catalogs?

Reply:
It's not needed to switch - I use it and another (nl.littlerobots.version-catalog-update) to handle dependency updates.